lib/google_search_data_viewer_worker/keywords/keyword_search_worker.ex

```diff
@@ -0,0 +1,22 @@
+defmodule GoogleSearchDataViewerWorker.Keywords.SearchWorker do
+  use Oban.Worker,
+    queue: :keyword_search,
+    max_attempts: 3,
+    unique: [period: 30]
+
+  alias GoogleSearchDataViewer.Keywords.GoogleSearchClient
+  alias GoogleSearchDataViewer.Keywords.Keyword
```

Comment:
```suggestion
  alias GoogleSearchDataViewer.Keywords.{GoogleSearchClient, Keyword}
```

Reply:
If I change this I get additional warnings regarding inconsistency; it states I have used multiple single line statements elsewhere, however I am unable to find these. 

Will look into it.

Reply:
I left it as it is for now as above.